.fixtures.yml

```diff
@@ -1,10 +1,12 @@
 fixtures:
   repositories:
-    'facts': 'git://github.com/puppetlabs/puppetlabs-facts.git'
-    'provision': "git://github.com/puppetlabs/provision.git"
-    'puppet_agent': 'git://github.com/puppetlabs/puppetlabs-puppet_agent.git'
-    'stdlib': "https://github.com/puppetlabs/puppetlabs-stdlib.git"
+    facts: 'https://github.com/puppetlabs/puppetlabs-facts.git'
+    provision: "https://github.com/puppetlabs/provision.git"
+    puppet_agent: 
+      repo: 'https://github.com/puppetlabs/puppetlabs-puppet_agent.git'
+      ref: v4.13.0
```

Comment:
why pin this module to a specific version?

Reply:
We opted to pin to v4.13.0 across our modules as it was the latest stable release that included puppet 8, and this is consistent and inline with the rest of our supported modules.
(and we preferred this approach rather than pulling from main)

Reply:
Why do you prefer pinning for this module, but not for the rest? Usually we try to run unit tests against master/main and pinned versions from metadata.json for acceptance tests.

Reply:
So we've had instances in our not so distant past where breaking changes were introduced in `main` of puppet_agent, and that turned our ci into a sea of red.
This is one way we've tried to limit that from occurring. 💯